src/util.rs

```diff
@@ -22,9 +22,19 @@ pub fn path_to_bytes<P: AsRef<Path>>(path: &P) -> Option<&[u8]> {
 }
 
 pub fn get_db() -> Result<DB> {
-    let mut path = config::zo_data()?;
-    path.push("db.zo");
-    DB::open(path)
+    let mut db_file = config::zo_data_dir()?;
+    db_file.push("db.zo");
+
+    // FIXME: fallback to old database location; remove in next breaking version
+    if let Some(mut old_db) = dirs::home_dir() {
```

Comment:
This just occurred to me - if the user already has a v0.3 database, it will get overwritten. I think we should only perform the migration if `db_file` is not a file.

Reply:
Good catch, thanks!

Reply:
Just as a note: you can't combine `if let` with a boolean, so that's why I had to check if `old_db` is a file and `db_file` is not a file.